main.go

```diff
@@ -0,0 +1,38 @@
+package main
+
+import (
+	"os"
+	"github.com/gorilla/pat"
+	"net/http"
+	"github.com/ONSdigital/dp-csv-splitter/handlers"
+	"github.com/ONSdigital/go-ns/log"
+	"github.com/ONSdigital/dp-csv-splitter/aws"
+)
+
+func main() {
+
+	bindAddr := os.Getenv("BIND_ADDR")
+	if len(bindAddr) == 0 {
+		bindAddr = ":21000"
+	}
+
+	fileProviderUrl := os.Getenv("FILE_PROVIDER_URL")
+	if len(fileProviderUrl) == 0 {
+		fileProviderUrl = ""
```

Comment:
This seems to me like it's saying `if the length of the string of fileProviderUrl is 0, set it to an empty string`. Seems a bit too much for me.